griptape/memory/structure/conversation_memory.py

```diff
@@ -16,6 +16,12 @@ class ConversationMemory:
     driver: Optional[BaseConversationMemoryDriver] = field(default=None, kw_only=True)
     runs: list[Run] = field(factory=list, kw_only=True)
     structure: Structure = field(init=False)
+    autoload = field(default=False, kw_only=True)
+
+    def __attrs_post_init__(self) -> None:
+        if self.driver and self.autoload:
+            autoloaded_memory = self.driver.load()
+            self.runs = autoloaded_memory.runs
```

Comment:
@vasinov should any of the other fields be loaded in?

Reply:
We should use `add_run`, so that before and after hooks get triggered correctly:

```python
[self.add_run(r) for r in self.driver.load().runs]
```

Reply:
We should use `add_run`, so that before and after hooks get triggered correctly:

```python
[self.add_run(r) for r in self.driver.load().runs]
```